SparseVector/SparseVector/SparseVector.cs

```diff
@@ -0,0 +1,119 @@
+﻿namespace SparseVector
+{
+    /// <summary>
+    /// class that implements sparse vector with addition, subtraction and
+    /// multiplication scalar operations
+    /// </summary>
+    public class SparseVector
+    {
+        private Dictionary<int, int> vector = new();
+        public int Size { get; private set; } = 0;
+
+        public SparseVector() { }
+
+        public SparseVector(Dictionary<int, int> other)
+        {
+            int maxIndex = 0;
+            foreach (var (index, value) in other)
+            {
+                if (index > maxIndex)
+                    maxIndex = index;
+                if (value != 0)
+                    vector[index] = value;
+            }
+            Size = maxIndex;
+        }
+
+        public SparseVector(int[] other)
+        {
+            Size = other.Length;
+            for (int i = 0; i < Size; i++)
+            {
+                if (other[i] != 0)
+                    vector[i] = other[i];
+            }
+        }
+
+        private void MultiplyByNegativeOne()
+        {
+            Dictionary<int, int> result = new();
+            foreach (var (index, elem) in vector)
+            {
+                result[index] = -elem;
+            }
+            vector = result;
+        }
+
+        private int GetValueAt(int index) => vector.GetValueOrDefault(index, 0);
+        private void AddValueAt(int index, int value)
```

Comment:
```suggestion
        private int GetValueAt(int index) => vector.GetValueOrDefault(index, 0);
        
        private void AddValueAt(int index, int value)
```
Надо пустую строчку между членами класса